Sources/NIOWebSocket/NIOWebSocketFrameAggregator.swift

```diff
@@ -0,0 +1,138 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2019 Apple Inc. and the SwiftNIO project authors
```

Comment:
Nit: can we update the copyright date here?

```suggestion
// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
```